docs/guide/content/user_guide/api_keys.md

```diff
@@ -0,0 +1,49 @@
+# API keys
+
+Some repository features in OnDemand Loop are only available to authenticated users. API keys let the application request restricted content — such as draft datasets or unpublished depositions — from services like Dataverse and Zenodo on your behalf.
```

Comment:
This is a bit misleading, it reads as if the user needs to login.
I think it should focus on content not users, eg:
`Some repository features in OnDemand Loop require access to restricted content`

Reply:
done